tests/test_filter_jobs.py

```diff
@@ -11,10 +11,16 @@
 def test_filter_jobs(base_url, selenium):
     """Open resultset page and filter for platform"""
     page = TreeherderPage(selenium, base_url).open()
-    platform = u'Linux'
+    all_builds = page.all_builds
 
+    platform = u'Linux'
     page.filter_by(platform)
-    assert platform in page.result_sets[0].builds[0].platform_name
+    filtered_builds = page.all_builds
+
+    buildname = page.result_sets[0].builds[0].platform_name
+    assert platform in buildname
+
+    assert all_builds > filtered_builds
```

Comment:
@davehunt, I tried this method and it failed as the len of all_builds equalled filtered_builds. Is there really any value of doing a count when the filter method does not change the number of result sets? To test that the filter works is much more dependent on the value of the build platform name rather than the count.

I've tried to think of different ways of counting the page values before and after a filter- and there's no guarantee that any value will be greater or equal to the original.